pkg/util/log/clog.go

```diff
@@ -930,6 +931,16 @@ func (sb *syncBuffer) rotateFile(now time.Time) error {
 		return err
 	}
 
+	// Redirect stderr to the current INFO log file in order to capture panic
```

Comment:
Yes, it is a little odd to use INFO here. FATAL would be better, but we don't create the file until the first write to it so we effectively can't use WARNING, ERROR or FATAL. I'm having deja-vu regarding your comment about multi-file logging output. I think we had this same discussion at Square. The reason for the multi-file-style logging is that the higher severities get rotated less frequently. Not sure how much that matters for us. I'd be fine with paring this down to a single log file. I'm not eager to add a fourth log file type.